.github/workflows/dev-astro.yml

```diff
@@ -10,7 +10,7 @@ env:
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ["development"]
+    branches: ["release/1.53.0"]
```

Comment:
Should this maybe be `release/**` so we don't have to keep changing it?

Reply:
We have multiple `release/**` branches currently out there and can't guarantee there won't ever be, so they might clobber each other.

Now we could do so fancy pants k8s deploy and dns magic on a per-branch if we wanted to get around that, but for now I say we just pick one.

Reply:
Yeah let's leave it as is then. I don't have time to work out the fancy magic right now.